addon/types/index.ts

```diff
@@ -4,7 +4,7 @@ export type Token = unknown;
 export interface IWaiter {
   name: WaiterName;
   waitUntil(): boolean;
-  debugInfo(): unknown;
+  debugInfo(): ITestWaiterDebugInfo[];
```

Comment:
I'd rather the generic thing not be forced into the same structure as what TestWaiter would use.

Reply:
I thought about this too, but from a consumer standpoint, we expect to output this debug info as part of `@ember/test-helpers`. Specifically, _that_ library _does_ expect this to be a particular format. In fact, it exactly expects this format, as demonstrated [here](https://github.com/emberjs/ember-test-helpers/blob/master/addon-test-support/%40ember/test-helpers/-internal/debug-info.ts#L157). How would we handle this otherwise?

Reply:
Kk, seems good then. Thank you!